drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -19,7 +19,26 @@ def self.options_for_type(pick_list_type, user:)
       case pick_list_type
       when 'COC'
         ::HUD.cocs_in_state(ENV['RELEVANT_COC_STATE']).sort.map do |code, name|
-          { code: code, label: name, secondary_label: code }
+          { code: code, label: "#{code} - #{name}" }
+        end
+
+      when 'STATE'
+        states = JSON.parse(File.read('drivers/hmis/lib/pick_list_data/states.json'))
+        states.map do |obj|
+          {
+            code: obj['abbreviation'],
+            label: "#{obj['abbreviation']} - #{obj['name']}",
+            initial_selected: obj['abbreviation'] == ENV['RELEVANT_COC_STATE'],
+          }
+        end
+
+      when 'GEOCODE'
+        geocodes = JSON.parse(File.read("drivers/hmis/lib/pick_list_data/geocodes/geocodes-#{ENV['RELEVANT_COC_STATE']}.json"))
+        geocodes.map do |obj|
```

Comment:
Probably not ideal to do this every time, we only need to load it into memory once. @eanders do you have a suggestion on how to cache here, given that its a class method?